### DISCUSSION TITLE
Including foundry fuzzed files for coverage, without duplicating tests and in reasonable time.

### DISCUSSION THREAD

**Discussion by smitrajput** [sablier-labs/evm-monorepo#698] [Q&A] [answered]
We referenced sabllier-v2's config files for our CI config, and noticed you didn't include fuzz files for coverage, and i assume you faced the same issue as us, of facing too much delay for running coverage for them on CI:
https://github.com/sablier-labs/v2-core/blob/main/.github/workflows/ci.yml#L256

We were thinking of ways to cover fuzzed files in CI with runs = 1 in foundry.toml, but that takes forever too, were you able to find some way to do that in CI? or you just copy their unit variant in the non-fuzzed section of your files to be able to run coverage for them successfully? 

The latter results in a lot of redundant test files, hence was wondering if you knew some way to do that

**Comment by smitrajput** [comment, marked as answer]
Update: putting runs = 1 in foundry.toml did end up working out :))

**Reply by PaulRBerg** [reply]
Glad to hear!

**Comment by PaulRBerg** [comment]
> or you just copy their unit variant in the non-fuzzed section of your files to be able to run coverage for them successfully?

This is exactly what we wound up doing.

> The latter results in a lot of redundant test files

Are they redundant? I find it helpful to have both simple tests that show how that your functions work with specific, hard-coded values, and other fuzz tests.
